sccp/figures/figureThomsonReduce.py

```diff
@@ -1,63 +0,0 @@
-"""Downsample Thomson and see how much the factors change.
-The goal here is to see how much the dataset size matters."""
-import scanpy as sc
-from .common import subplotLabel, getSetup
```

Comment:
Should this be removed? I thought @bremigio is adding to this on another branch?

Reply:
Yes I thought I removed this file, it shows on my end that I did. Let me double check.